pubtools/pulplib/_impl/model/repository/yum.py

```diff
@@ -70,6 +70,18 @@ class YumRepository(Repository):
 
     type = pulp_attrib(default="rpm-repo", type=str, pulp_field="notes._repo-type")
 
+    population_sources = pulp_attrib(
+        default=(), type=tuple, converter=tuple, pulp_field="notes.population_sources"
+    )
+    """List of repositories used for populate the repository
+    """
+
+    ubi_population = pulp_attrib(
+        default=False, type=bool, pulp_field="notes.ubi_population"
+    )
+    """Flag indicating whether repo should be populated or not
```

Comment:
population_sources gives the impression of something a bit generic but this one is named specifically relating to UBI.  I guess the doc should be slightly more detailed then, like:

```suggestion
    """Flag indicating whether repo should be populated or not for the purposes of UBI
```